ansible/roles/contrail-ansible-deployer/tasks/main.yaml

```diff
@@ -29,6 +29,8 @@
     ANSIBLE_HOST_KEY_CHECKING: 'False'
     ANSIBLE_STDOUT_CALLBACK: debug
   become: True
+  tags:
+    - skip_ansible_lint # Always run internal playbook
```

Comment:
Could you elaborate on reason for skipping ansible lint here in comment? Like: linter wants X, but we have to Y.

(in other places as well)